YAML/tests/syntax_test_block.yaml

```diff
@@ -18,11 +18,11 @@
 
 - >1
 # ^ keyword.control.flow.block-scalar
-#  ^ constant.numeric.indentation-indicator
+#  ^ constant.numeric.integer.decimal.indentation-indicator
```

Comment:
Can you expand upon why this punctuation is part of the integer literal?

It is probably just that even after reading the YAML spec and this, I have no idea what the indentation-indicator does/is.

Reply:
Actually, I didn't see it before as well. We might ask @FichteFoll for it.

It just intents to follow _Bash_ which scopes numeric file descriptors like `constant.numeric.integer.decimal.file-descriptor.shell`.

... just to ensure it gets highlighted as integer if someone will ever address `constant.numeric.integer.decimal` in a color scheme.

Otherwise I would suggest something like `constant.other` or `constant.language` as we did with the hash in the git syntaxes.

Reply:
Oh, sorry, I guess I misread what was getting the `indentation-indicator`. I now see it only applies to the `1` and the `indentation-indicator` is just a specialization. I thought it was applying to the `>`.

Reply:
`>` introduces a folded block scalar. The `1` controls the indentation of the content; it isn't part of the content. (This particular block scalar doesn't actually have any content; it ends at the beginning of the next line when it sees the `#`.)